### PR TITLE
ppj08

### PR DESCRIPTION
**Pull request by valeria1604**

### CODE REVIEW

File: src/com/company/ppj08/Tasks123.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        do {
+            System.out.println(i++);
+        } while (i < 5);
+    }
+
+    private static void task3_example2() {
+        int answer;
+        do {
+            System.out.println("What is 2+2?");
+            Scanner scan = new Scanner(System.in);
```
Comment (maxline) [inline src/com/company/ppj08/Tasks123.java]: please do not create new Scanner instance inside the loop. Move it to begin of the method.

File: src/com/company/ppj08/Tasks45.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        task5();
+    }
+
+    private static void task4() {
+        int day = 4;
+        int month = 4;
+        int countDays = 0;
+        for (int numberMonth = 1; numberMonth < month; numberMonth++) {
+            countDays += 31;
+            if (numberMonth == 2) {
```
Comment (maxline) [inline src/com/company/ppj08/Tasks45.java:18]: Why we check that numberMonth equals only `2`, the program should return correct result for any data in any month during the ear. Please finish the solution.

File: src/com/company/ppj08/Tasks45.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        }
+        System.out.println(countDays + day);
+    }
+
+    private static void task5() {
+        int countRows = 5;
+        int countColumns = 8;
+        for (int rows = 1; rows <= countRows; rows++) {
+            for (int columns = 1; columns <= countColumns; columns++) {
+                if (rows % 2 == 0) {
```
Comment (maxline) [inline src/com/company/ppj08/Tasks45.java]: good solution for the first variant. let's try to simply `if else` costruction, try to not use inner `if` blocks

File: src/com/company/ppj08/Tasks45.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+                        System.out.print("x");
+                    }
+                } else {
+                    if (columns % 2 == 0) {
+                        System.out.print("x");
+                    } else {
+                        System.out.print("o");
+                    }
+                }
+                if (columns == countColumns) {
```
Comment (maxline) [inline src/com/company/ppj08/Tasks45.java]: do we really need this `if` block?

**Review by maxline — CHANGES_REQUESTED** [review]
Good job! let' improve the solution

File: src/com/company/ppj08/Tasks123.java (hunk 1 above)
Comment (valeria1604) [reply src/com/company/ppj08/Tasks123.java]: fixed

File: src/com/company/ppj08/Tasks45.java (hunk 2 above)
Comment (valeria1604) [reply src/com/company/ppj08/Tasks45.java:18]: fixed

**Review by maxline — APPROVED** [review]
approved

**Pull request merged**